src/js/page.js

```diff
@@ -29,6 +29,7 @@ export default {
      * @type {Function}
      */
     init() {
+        document.querySelectorAll( '[tabindex="0"]' ).forEach( el => el.setAttribute( 'tabIndex', -1 ) );
```

Comment:
I don't feed good about this kind of DOM correction. Makes more sense to fix at the source, or am I overlooking something?

Reply:
it seems this line isn't needed after all; after testing these changes again with a screen reader, it seems focus stays were it's expected to without this line